core/src/main/java/uk/ac/bbsrc/tgac/miso/core/data/AbstractSequencerPartitionContainer.java

```diff
@@ -194,13 +183,32 @@ public void setPaired(Boolean paired) {
   public abstract T getPartitionAt(int partitionNumber);
 
   @Override
-  public Run getRun() {
-    return run;
+  public Collection<Run> getRuns() {
+    return runs;
+  }
+
+  @Override
+  public void setRuns(Collection<Run> runs) {
+    this.runs = runs;
+  }
+
+  @Override
+  public Run getLastRun() {
+    Run lastRun = null;
+    for (Run thisRun : getRuns()) {
+      if (thisRun == null) continue;
+      if (lastRun == null) {
+        lastRun = thisRun;
+      } else if (thisRun.getStatus() != null && thisRun.getStatus().getStartDate().after(lastRun.getStatus().getStartDate())) {
```

Comment:
If there are two runs and neither has a status, this will return whichever was examined first - likely the earliest run assuming the collection will be ordered by natural DB order